### ISSUE TITLE
Enable 'style'-support in version 1.x

### ISSUE THREAD

**Issue by freakkiller** [cascornelissen/svg-spritemap-webpack-plugin#42]
We love your plugin. Thanks for programming. Unfortunately, we can not yet upgrade our projects to the current Webpack version. Therefore we have to use plugin version 1.x, which doesn't yet support the 'style' option.. Is it possible to integrate the style support in this version as well?

**Comment by cascornelissen** [comment]
I tried implementing it when webpack 4 wasn't out yet but couldn't get it to work without an immense amount of work. I then decided to wait for webpack 4 with its improved plugin system and that made it a lot easier to implement.

I'm not saying it's impossible to do but it will be quite hard to do correctly and I don't have the time (or energy for that matter) for it but everyone is free to submit a PR. A related problem is that the `styles` option is quite complex because it needs to support multiple languages (the newly introduced [variables setup](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/blob/v2.6.0/docs/variables.md) makes it even more complex) which means a solid test suite is pretty much required at this point.

Related: https://github.com/cascornelissen/svg-spritemap-webpack-plugin/issues/26#issuecomment-386513623
